dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/user/UserController.java

```diff
@@ -153,6 +153,18 @@
             params.setCanManage( true );
             params.setAuthSubset( true );
         }
+        
+        if ( !filters.isEmpty() )
```

Comment:
Perhaps simplify with this?

```
params.setPrefetchUserGroups( filters.stream()
    .anyMatch( f -> f.startsWith( "userGroups." ) ) );
```

Reply:
Or even make a utility method somewhere in the web api called `hasFilter( List<String> filters )`?

Maybe put in `DefaultContextService` or in some util class. I think we are doing this type of checking multiple places and it would be nice to encapsulate, centralize and document it.